core/platform/auth/firebase_auth_services.py

```diff
@@ -60,16 +60,25 @@
 
 import firebase_admin
 from firebase_admin import auth as firebase_auth
+from firebase_admin import exceptions as firebase_exceptions
 
 (auth_models,) = models.Registry.import_models([models.NAMES.auth])
 
+firebase_auth.ImportUserRecord('f')
```

Comment:
What is this?

Reply:
LOL whoops, nice catch! This was how I found my way into the firebase SDK's internals... Fixed..!